tests/golden/good/2.6-word-expansions/2.6.2-parameter-expansion/word-following-variable-parameters.t/expected.json

```diff
@@ -483,7 +483,7 @@
                                           "'}'",
                                           [
                                             [
-                                              "WordDoubleQuoted",
```

Comment:
I don't think this one is correct because now we have a `WordSingleQuoted` inside another `WordSingleQuoted`. I think that, similarly to the other cases, we should just remove the layer of `WordDoubleQuoted` altogether. Not entirely sure though.